src/Illuminate/Cache/Repository.php

```diff
@@ -175,9 +177,11 @@ protected function handleManyResult($keys, $key, $value)
     /**
      * Retrieve an item from the cache and delete it.
      *
+     * @template TCached of mixed
```

Comment:
Please use `TCacheValue` everywhere, and remove `of mixed`.

Reply:
Oh neat I wasn't aware of that short hand